.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ on:
     - cron: '0 */12 * * *'
 
 env:
-  GO_VERSION: 1.15.5
+  GO_VERSION: 1.16
```

Comment:
should we add the minor to have more control?

Reply:
Do we care? To me, pinning minor version is just a maintenance headache.

Reply:
golang changes affect projects, most recently example is one that changed null detection/validation, it broke kubernetes  and calico

https://github.com/kubernetes/kubernetes/issues/99804
https://github.com/projectcalico/calico/issues/4443

We should have full control of the CI environment so we can detect and track these changes, this way, if we bump manually from 1.16.2 to 1.16.3 and it fails we know that the golang version bump is the problem.
Otherwise, we'll notice that it "suddenly" started to fail and some hours/days later of precious engineer time someone will notice, hey the golang version is different
It also allow us to say, this was tested with golang 1.16.2, ovn 2.13.2, ovs x.y.z, ... that is important, because we had the same issue with ovn/ovs, where new versions broke CI

Reply:
Right, but that's a switch between minor versions. If we think there will be a regression in patch versions, I can go back to pinning that too. It does seem unlikely.

Reply:
One example with a patch release https://github.com/kubernetes/kubernetes/issues/75833#issuecomment-489885227

Reply:
fair 'nuff.